h2o-genmodel/src/main/java/hex/genmodel/algos/tree/ConvertTreeOptions.java

```diff
@@ -0,0 +1,23 @@
+package hex.genmodel.algos.tree;
+
+public class ConvertTreeOptions {
+  static final ConvertTreeOptions DEFAULT = new ConvertTreeOptions();
```

Comment:
I would very much like _checkTreeConsistency to be final, making this an immutable object, which in turn would make this static safe from modification

Reply:
that makes sense, good point @honzasterba - thank you